contrib/tools/meta.md/eqform.md

```diff
@@ -238,6 +256,21 @@ fmod EQFORM is
       sort NoFalseForm{Term} to NoFalseForm,
       sort NormForm{Term} to NormForm,
       sort Form{Term} to Form) .
+
+  vars T T' : Term . vars F F' : Form . var SUB : Substitution .
+
+  op _<<_ : Form Substitution -> [Form] .
+  ---------------------------------------
+  eq tt << SUB = tt .
+  eq ff << SUB = ff .
+
+  eq (T ?= T') << SUB = (T << SUB) ?= (T' << SUB) .
+  eq (T != T') << SUB = (T << SUB) != (T' << SUB) .
+
+  eq (~ F) << SUB = ~ (F << SUB) .
+
+ ceq (F /\ F') << SUB = (F << SUB) /\ (F' << SUB) if F =/= tt /\ F' =/= tt .
+ ceq (F \/ F') << SUB = (F << SUB) \/ (F' << SUB) if F =/= ff /\ F' =/= ff .
```

Comment:
@skeirik2 let's discuss here where it's easier to talk about the equations in question.

I think that the side-conditions are sufficient to assure that we don't have an infinite recursion here?

Reply:
Look good to me.